traceapp/tmpl.go

```diff
@@ -97,7 +98,12 @@ func (a *App) parseHTMLTemplates(sets [][]string) error {
 			"dict":              dict,
 		})
 		for _, tmp := range set {
-			tmplBytes, err := tmpl.Asset(tmp)
+			tmplFile, err := tmpl.Assets.Open("/" + tmp)
+			if err != nil {
+				return fmt.Errorf("template %v: %s", set, err)
+			}
+			tmplBytes, err := ioutil.ReadAll(tmplFile)
+			tmplFile.Close()
```

Comment:
As an observation, if this were my codebase, I would use a helper similar to `io.ReadFile` but for virtual filesystems:

[`vfsutil.ReadFile`](https://godoc.org/github.com/shurcooL/go/vfs/httpfs/vfsutil#ReadFile)

```Go
tmplBytes, err := vfsutil.ReadFile(tmpl.Assets, "/" + tmp)
...
```

Then the number of lines would not increase here. But I figured it's not worth pulling in an extra package just to replace a standard library package and 4 extra lines of standard code.